task/06-conditions-n-loops-tasks.js

```diff
@@ -46,7 +53,10 @@ function getFizzBuzz(num) {
  *   10 => 3628800
  */
 function getFactorial(n) {
-    throw new Error('Not implemented');
+    let result = 1;
+    while (n > 0) 
+        result *= n--;
```

Comment:
while (n > 0)  { 
        result *= n--;
    }

Sometimes it is better to use the  curly brackets in any case for loops  for better readability